File: Python Labs/com/pattern/builder/ClothingItem.py
class ClothingItem:
    def __init__(self, type, color, size):
        self.type = type
        self.color = color
        self.size = size

    def __str__(self):
        return f'ClothingItem [type: {self.type}, color: {self.color}, size: {self.size}]'

File: Java Labs/Patterns On Java/src/com/pattern/builder/ClothingItem.java
package com.pattern.builder;

public class ClothingItem {
    private String type;
    private String color;
    private int size;

    public ClothingItem(String type, String color, int size) {
        this.type = type;
        this.color = color;
        this.size = size;
    }

    public String getType() {
        return type;
    }

    public String getColor() {
        return color;
    }

    public int getSize() {
        return size;
    }

    @Override
    public String toString() {
        return "ClothingItem [type=" + type + ", color=" + color + ", size=" + size + "]";
    }
}
